.github/workflows/release.yml

```diff
@@ -0,0 +1,30 @@
+name: Release Pipeline
+
+on:
+  release:
+    types: [ published ]
+
+jobs:
+  npm-relase:
+    name: 'Release to npm'
+    if: ${{ startsWith(github.ref, 'refs/tags/') }}
```

Comment:
Also, since we are already using release trigger, this is not need right?

Reply:
Removed.